### ISSUE TITLE
Support for alpha transparency on the fill of eyeplot?

### ISSUE THREAD

**Issue by helske** [mjskay/tidybayes#163]
It seems that `geom_eye` doesn't support alpha transparency directly. I tried to hack around it by using `scale_alpha_manual` but the also the line range becomes transparent. Any change of adding support for alpha? Related to this, it would be nice to have an option to color the edges of the eyes as well in some cases.

**Comment by mjskay** [comment]
Definitely on both. These options will likely be coming when I rewrite those geoms this summer (see #84). I've added a note there so I remember to address this, so I'll close this for now.

In the mean time, here are some workarounds depending on what you're trying to do:

- If you're just trying to change the alpha and your fill color is not being mapped onto a column in the data (so the fill color is fixed), you can use `adjustcolor()` to get a version of the color with transparency. E.g. if you had something like `geom_eye(fill = "red")` you can replace it with something like `geom_eye(fill = adjustcolor("red", alpha.f = 0.5))`

- If you need finer-grained control, you can replace a `geom_eye` with a combination `geom_violin` and `stat_pointinterval` and then control the properties of the violin geom directly. E.g. this:

```r
ggplot(...) +
  geom_eye()
```

Becomes this:

```r
ggplot(...) +
  geom_violin() +
  stat_pointinterval()
```

Then you can set `alpha` on the `geom_violin()`.

Similar combinations work for `geom_eyeh()` and `geom_halfeyeh()` (though for the latter you would use `ggridges::geom_density_ridges()` instead of a violin).

**Comment by mjskay** [comment]
If you're curious, this should be fixed in the next release via #84 (currently the fixed version is on the `meta_geom` branch).
